duckduckgo/duckduckgo-autofill
# Fix extension bugs

**Reviewer:** @shakyShane @alistairjcbrown 
**Asana:** https://app.asana.com/0/1200930669568058/1205206733038177/f

## Description
- Private address not refreshing
- Autofill buttons sometimes not clickable
- Fix flytap username

## Steps to test
**Please do test this as I'm having trouble with private address generation and need further validation.**
- Visit https://gravida.pro/ddg/blob-test.html from Chrome
- Fill in a private address
- Fill in another one
- → Ensure that the field is filled and that at the second attempt you are provided with a different private address
